### PR TITLE
Add support lang

### PR DESCRIPTION
**Pull request by miki-matsumoto**

### CODE REVIEW

**Comment by toinoudivad** [comment]
Please fix the code and tests, looks like it's breaking a lot of things

File: lib/wovnrb/lang.rb [hunk 1]
```diff
@@ -25,16 +26,19 @@ class Lang
...
       'pt' => {name: 'Português',         code: 'pt',     en: 'Portuguese'},
       'ru' => {name: 'Русский',           code: 'ru',     en: 'Russian'},
       'es' => {name: 'Español',           code: 'es',     en: 'Spanish'},
+      'es' => {name: 'Kiswahili',         code: 'sw',     en: 'Swahili'},
       'sv' => {name: 'Svensk',            code: 'sv',     en: 'Swedish'},
       'th' => {name: 'ภาษาไทย',           code: 'th',     en: 'Thai'},
       'hi' => {name: 'हिन्दी',               code: 'hi',     en: 'Hindi'},
       'tr' => {name: 'Türkçe',            code: 'tr',     en: 'Turkish'},
       'uk' => {name: 'Українська',        code: 'uk',     en: 'Ukrainian'},
+      'uk' => {name: 'اردو',              code: 'ur',     en: 'Urdu'},
```
Comment (a-bougouin) [inline lib/wovnrb/lang.rb]: I think the key should be `ur`, not `uk`.

File: lib/wovnrb/lang.rb [hunk 2]
```diff
@@ -25,16 +26,19 @@ class Lang
       'ms' => {name: 'Bahasa Melayu',     code: 'ms',     en: 'Malay'},
       'my' => { name: 'ဗမာစာ',             code: 'my',     en: 'Burmese' },
       'ne' => {name: 'नेपाली भाषा',            code: 'ne',     en: 'Nepali'},
+      'no' => {name: 'زبان_فارسی',        code: 'fs',     en: 'Persian'},
       'no' => {name: 'Norsk',             code: 'no',     en: 'Norwegian'},
       'pl' => {name: 'Polski',            code: 'pl',     en: 'Polish'},
       'pt' => {name: 'Português',         code: 'pt',     en: 'Portuguese'},
       'ru' => {name: 'Русский',           code: 'ru',     en: 'Russian'},
       'es' => {name: 'Español',           code: 'es',     en: 'Spanish'},
+      'es' => {name: 'Kiswahili',         code: 'sw',     en: 'Swahili'},
```
Comment (a-bougouin) [inline lib/wovnrb/lang.rb]: I think the key should be `sw`, not `es`.

File: lib/wovnrb/lang.rb [hunk 3]
```diff
@@ -25,16 +26,19 @@ class Lang
       'ms' => {name: 'Bahasa Melayu',     code: 'ms',     en: 'Malay'},
       'my' => { name: 'ဗမာစာ',             code: 'my',     en: 'Burmese' },
       'ne' => {name: 'नेपाली भाषा',            code: 'ne',     en: 'Nepali'},
+      'no' => {name: 'زبان_فارسی',        code: 'fs',     en: 'Persian'},
```
Comment (a-bougouin) [inline lib/wovnrb/lang.rb]: I think this key should be `fs`, not `no`.

**Pull request merged**